### DISCUSSION TITLE
Modals vs Toast for Success Messages

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#2846] [Q&A] [answered]
Hi,

Is there any specific guidance around whether to use a modal or a toast to indicate success after a user takes an action through a modal (e.g. User enters data on a modal to update their support plan and then clicks on Submit)?

**Comment by SiTaggart** [comment, marked as answer]
Hi @tonge495 

Have you read the pattern guidelines on notifications and feedback indicators? https://paste.twilio.design/patterns/notifications-and-feedback#feedback-indicators

We go into some detail there on when to use each one. Let us know if thats what you're looking for.

**Reply by tonge495** [reply]
Got it thank you!
